vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -598,6 +600,17 @@ class RoomDetailFragment @Inject constructor(
         }
     }
 
+    private fun handleShowLocationPreview(viewEvent: RoomDetailViewEvents.ShowLocation) {
+        navigator
+                .openLocationSharing(
+                        context = requireContext(),
+                        roomId = roomDetailArgs.roomId,
```

Comment:
Do we need the `roomId` in the mode LocationSharingMode.PREVIEW?

Reply:
Not for now, but probably will need to stop live location sharing.